### PR TITLE
Setup database and created tests for server.

### PR DESCRIPTION
**Pull request by asownder95**

### CODE REVIEW

**Comment by asownder95** [comment]
@aliu730 Hey Andrew! Can you review my PR? I forgot to attach Trello ticket to this so I'm attaching it below. Thanks!

[Trello Ticket](https://trello.com/c/BXSvUkSt/14-overviewmod-create-test-for-server-and-database-s)

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const db = require('../database/index');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../public')));
+// app.use(bodyParser.json());
```
Comment (aliu730) [inline server/app.js:9]: Should be used above line 8 if used. Remove comment.

**Review by aliu730 — APPROVED** [review]
Looks great, nice test coverage as well.

**Review by aliu730 — APPROVED** [review]
Looks good, great test coverage as well.

File: server/app.js (hunk 1 above)
Comment (asownder95) [reply server/app.js:9]: Nice catch, accidentally commented that out. Will fix that and re-commit.

**Pull request merged**